pipeline/07-export.R

```diff
@@ -365,22 +413,136 @@ assessment_card_prepped <- assessment_card %>%
 for (town in unique(assessment_pin_prepped$township_code)) {
   message("Now processing: ", town_convert(town))
 
-
-  ## 5.1. PIN-Level ------------------------------------------------------------
+  workbook_name <- glue(
+    params$assessment$year,
+    str_replace(town_convert(town), " ", "_"),
+    "Initial_Model_Values.xlsx",
+    .sep = "_"
+  )
 
   # Filter overall data to specific township
   assessment_pin_filtered <- assessment_pin_prepped %>%
     filter(township_code == town) %>%
     select(-township_code)
 
+  # 5.3 Comp details -----------------------------------------------------------
+
+  training_data_filtered_1 <- training_data %>%
+    inner_join(
+      assessment_pin_filtered %>% select(comp_pin_1),
+      by = join_by(meta_pin == comp_pin_1)
+    )
+
+  training_data_filtered_2 <- training_data %>%
+    inner_join(
+      assessment_pin_filtered %>% select(comp_pin_2),
+      by = join_by(meta_pin == comp_pin_2)
+    )
+
+  training_data_filtered <- bind_rows(
+    training_data_filtered_1, training_data_filtered_2
+  ) %>%
+    unique()
+
+  training_data_selected <- training_data_filtered %>%
+    select(
+      meta_pin, meta_sale_price, meta_sale_date, meta_class, meta_nbhd_code,
+      char_yrblt, loc_property_address, char_beds, char_ext_wall,
+      char_heat, char_bldg_sf, char_type_resd, char_land_sf
+    ) %>%
+    ccao::vars_recode(type = "long")
+
+  comp_sheet_name <- "Sales"
```

Comment:
Something like `Sale Detail` would be clearer, but it seemed like within-workbook links only worked if the sheet name had no spaces. Kind of an annoying limitation.

Reply:
Oops! Disregard my earlier comment then. Maybe `Comparables`?

Reply:
I like that, done in e02cdd29b4cc03e629f9f0abeff53a51fccb632b.